statsmodels/survey/examples/survey_df.csv

```diff
@@ -0,0 +1,184 @@
+"dnum","pw","api00","api99"
```

Comment:
Fair enough.  But my suggestion for Jarvis is to start with very simple datasets that can be inlined, e.g.

```
data = np.r_[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
cluster = np.r_[0, 0, 1, 1, 2, 2, 3, 3, 3, 4, 4, 4]
strata = np.r_[0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1]
```

then move onto larger datasets that can be loaded from files.